CI/travis.linux.after_success.sh

```diff
@@ -6,11 +6,11 @@ if [[ "${MUDLET_VERSION_BUILD}" == -ptb* ]]; then
   public_test_build="true"
 fi
 
-# we deploy only linux+deploy or cron+clang+cmake for PTB
-if { [ "${TRAVIS_OS_NAME}" = "linux" ] && [ "${DEPLOY}" = "deploy" ]; } ||
-   { [ "${TRAVIS_EVENT_TYPE}" = "cron" ] && [ "${TRAVIS_OS_NAME}" = "linux" ] &&  [ "${CC}" = "clang" ] && [ "${Q_OR_C_MAKE}" = "cmake" ]; } then
+# we deploy only deploy or cron+clang+cmake for PTB
+if { [ "${DEPLOY}" = "deploy" ]; } ||
```

Comment:
This logic should not be getting changed in this PR, the scope has nothing to do with it.

Reply:
Sorry, but it was such a glaring bit of bogus code that I could resist it. Will move to another PR.